pkg/commands/serviceversion/activate.go

```diff
@@ -53,6 +54,7 @@ func NewActivateCommand(parent argparser.Registerer, g *global.Data) *ActivateCo
 // Exec invokes the application logic for the command.
 func (c *ActivateCommand) Exec(_ io.Reader, out io.Writer) error {
 	serviceID, serviceVersion, err := argparser.ServiceDetails(argparser.ServiceDetailsOpts{
+		Active:             optional.Of(false),
```

Comment:
This is one of the actual bug fix changes: `fastly service-version activate` will now fail if the specified service-version is active, but will not fail if it is locked.